docs/diagrams/EditSequenceDiagram.puml

```diff
@@ -0,0 +1,69 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":InternBuddyParser" as InternBuddyParser LOGIC_COLOR
+participant ":EditCommandParser" as EditCommandParser LOGIC_COLOR
+participant "e:EditCommand" as EditCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute("edit 2 n/Google")
+activate LogicManager
+
+LogicManager -> InternBuddyParser : parseCommand("edit 2 n/Google")
+activate InternBuddyParser
+
+create EditCommandParser
+InternBuddyParser -> EditCommandParser
+activate EditCommandParser
+
+EditCommandParser --> InternBuddyParser
+deactivate EditCommandParser
+
+InternBuddyParser -> EditCommandParser : parse("2 n/Google")
+activate EditCommandParser
+
+create EditCommand
+EditCommandParser -> EditCommand
+activate EditCommand
+
+EditCommand --> EditCommandParser : d
+deactivate EditCommand
+
+EditCommandParser --> InternBuddyParser : d
+deactivate EditCommandParser
+'Hidden arrow to position the destroy marker below the end of the activation bar.
+EditCommandParser -[hidden]-> InternBuddyParser
+destroy EditCommandParser
+
+InternBuddyParser --> LogicManager : d
+deactivate InternBuddyParser
+
+LogicManager -> EditCommand : execute()
+activate EditCommand
+
+EditCommand -> Model : setInternship(toEdit, edited)
+activate Model
+
+Model --> EditCommand
+deactivate Model
+
+create CommandResult
+EditCommand -> CommandResult
+activate CommandResult
+
+CommandResult --> EditCommand
+deactivate CommandResult
+
+EditCommand --> LogicManager : result
```

Comment:
Is there a reason why the `EditCommand` has no cross at the end of its activation bar?

Reply:
Actually Im not too sure when to put the cross haha
The deletecommand in the ab3 developer guide dosent have a cross